ReSharper.FSharp/src/FSharp/FSharp.Psi/src/FSharp.psi

```diff
@@ -423,6 +423,10 @@ interface fSharpExpression:
   libraryOnlyExpr |
   fromErrorExpr;
 
+interface lambdaLikeExpr:
+  lambdaExpr |
+  dotLambdaExpr;
```

Comment:
What getters and logic do these interfaces share in common? It seems there's not much actual intersection?